languages.toml

```
@@ -2341,3 +2341,19 @@ indent = { tab-width = 2, unit = "  " }
 [[grammar]]
 name = "capnp"
 source = { git = "https://github.com/amaanq/tree-sitter-capnp", rev = "fc6e2addf103861b9b3dffb82c543eb6b71061aa" }
+
+[[language]]
+name = "smithy"
+scope = "source.smithy"
+injection-regex = "smithy"
+file-types = ["smithy"]
+shebangs = []
+roots = ["smithy-build.json"]
+auto-format = false
+comment-token = "//"
+indent = { tab-width = 4, unit = "  "  }
```

```suggestion
indent = { tab-width = 4, unit = "    "  }
```

From the docs it looks like this should be 4 columns, right?